libs/mark-flow-ui/src/pages/contest_page.tsx

```diff
@@ -92,39 +104,81 @@ export function ContestPage(props: ContestPageProps): JSX.Element {
     }
   })();
 
+  const nextContestButtonRef = useRef<Button<never>>(null);
+  const handleOnPressNext = useCallback((event: ButtonPressEvent) => {
+    if (
+      getInteractionMethod(event) ===
+      VoteUpdateInteractionMethod.AssistiveTechnologyDevice
+    ) {
+      // When navigating by accessible device, need to blur or the focus
+      // on "Next" button will be retained for next contest,
+      // increasing chance the voter will accidentally skip a contest
+      nextContestButtonRef?.current?.blur();
+    }
+  }, []);
+
   const nextContestButton = (
     <LinkButton
       id="next"
       rightIcon="Next"
       variant={isVoteComplete ? 'primary' : 'neutral'}
+      onPress={handleOnPressNext}
```

Comment:
It feels a little weird to me that LinkButton now takes an optional `onPress` callback that can get called, because it's not immediately clear how that interacts with the actual navigation logic (which one run firsts? does `onPress` override `to` or complement it?)

A few alternative ideas:
- Potentially using a `key` prop with the contest id somewhere would force React to rerender the elements rather than reuse the existing elements (e.g. on this button, or on the whole screen - not sure exactly where the right place is)
- Switch from `LinkButton` back to `Button`. I know I suggested sticking with `LinkButton` in the first place, but I didn't realize the context of where we wanted to call `blur` from. Sorry for not reading more closely!

Reply:
I see the argument; I had initially called the prop `additionalOnPressBehavior` but thought it introduced more confusion than clarity. Switched to `key` in [d17ac75](https://github.com/votingworks/vxsuite/pull/4284/commits/d17ac7555b4b8ca9cfbf4dccd07219f13cd9e8f9).

This also revealed a rough edge where focus resumes at the "Next" button, instead at the top of the page, when advancing from one contest to another:

https://github.com/votingworks/vxsuite/assets/1060688/bc2a5966-5f8e-4640-b43e-bc3824e6aa62

This is fixable by adding an `autoFocus` prop to the various contest types and `ContestChoiceButton` but I think we should actually piggyback off the `id="audiofocus"` mentioned above: https://github.com/votingworks/vxsuite/pull/4284#discussion_r1431641061

Can tackle this in another PR, since this one is dragging out.

Reply:
Nice catch on the Next button focus issue. I agree addressing in a follow up makes sense.

Another potential way to handle that would be to have a child component for the contest screen (including buttons) for an individual contest that is rendered with `key={contest.id}`. Would that reset the focus state for the whole screen each time you move between contests? If so, that would also solve the link button focus issue from earlier in this thread.